### DISCUSSION TITLE
How to enable custom code for the sections of the donation selection

### DISCUSSION THREAD

**Discussion by OliHS** [jonas-hoebenreich/wc-donation-platform#50] [Q&A] [answered]
In the documentation : Advanced Layout (Expert: Custom Code with action…), it say "When editing a product, you can enable custom code for the sections of the donation selection form". But I can't find how to do it . Any help please? Many thanks
![image](https://github.com/jonas-hoebenreich/wc-donation-platform/assets/68728362/b3aca883-c57e-4e86-99e9-d5d7178d69e5)

**Comment by jonas-hoebenreich** [comment, marked as answer]
Hi OliHS,

have you followed this guide? https://wcdp.jonh.eu/documentation/advanced/advanced-layout-expert-custom-code-with-action/
Please note: This functionality is designed for proficient and advanced users with a strong command of PHP, Javascript, and HTML programming. If the provided guide doesn’t assist you in implementing custom actions, this feature may not be suitable for your expertise level.
Given that this feature caters to expert users, I’m unable to offer complimentary support for it.

Best,
Jonas
